airbyte-db/src/main/java/io/airbyte/db/instance/BaseDatabaseInstance.java

```diff
@@ -54,18 +54,20 @@
    *        jdbc:postgresql://${DATABASE_HOST}:${DATABASE_PORT/${DATABASE_DB}
    * @param databaseName this name is only for logging purpose; it may not be the actual database name
    *        in the server
+   * @param isDatabaseReady a function to check if the database has been initialized and ready for
+   *        consumption
    */
   protected BaseDatabaseInstance(String username,
                                  String password,
                                  String connectionString,
-                                 String schema,
+                                 String initialSchema,
                                  String databaseName,
                                  Set<String> tableNames,
                                  Function<Database, Boolean> isDatabaseReady) {
     this.username = username;
     this.password = password;
     this.connectionString = connectionString;
-    this.schema = schema;
+    this.initialSchema = initialSchema;
```

Comment:
`schema` is a loaded term. It may mean Postgres named schema, or table structure. By renaming it to initial schema, it becomes obvious that it means the latter (since Postgres named schema is just a name and does not change). Also by adding `initial`, it emphasize that it is not the final schema.